core/kazoo_media/src/kz_media_tts_cache.erl

```diff
@@ -171,8 +171,13 @@ handle_info({'http', {ReqID, 'stream', Bin}}, #state{kz_http_req_id=ReqID
                                    ,timer_ref=start_timer()
                                    }};
         <<"application/json">> ->
+            lager:debug("JSON response: ~s", [Bin]),
+            {'noreply', State, 'hibernate'};
+
+        <<"application/json; charset=UTF-8">> ->
```

Comment:
you can change this to one clause with  `<<"application/json", _/binary>>` tho is strange the content-type comes that way

Reply:
Better would be to use something like we did in [crossbar](https://github.com/2600hz/kazoo/blob/master/applications/crossbar/src/modules/cb_modules_util.erl#L105) and parse the header. You get back something like `{{<<"application">>, <<"json">>, '*' | proplist()}, _Weight, _Extensions}`